internal/pkg/describe/app_test.go

```diff
@@ -78,8 +78,8 @@ func TestAppDescriber_EnvVars(t *testing.T) {
 				)
 			},
 			wantedEnvVars: map[string]string{
-				"ECS_CLI_APP_NAME":         "my-app",
-				"ECS_CLI_ENVIRONMENT_NAME": "prod",
+				"COPILOT_SERVICE_NAME":     "my-app",
```

Comment:
nit: can we call this my-service, instead? 
```suggestion
				"COPILOT_SERVICE_NAME":     "my-service",
```

Reply:
yeah I can make this change. But I was planning to fix this test file when doing renaming for `describe` pkg.